students/hyeonjin/westagram/urls.py

```diff
@@ -0,0 +1,21 @@
+"""westagram URL Configuration
+
+The `urlpatterns` list routes URLs to views. For more information please see:
+    https://docs.djangoproject.com/en/4.0/topics/http/urls/
+Examples:
+Function views
+    1. Add an import:  from my_app import views
+    2. Add a URL to urlpatterns:  path('', views.home, name='home')
+Class-based views
+    1. Add an import:  from other_app.views import Home
+    2. Add a URL to urlpatterns:  path('', Home.as_view(), name='home')
+Including another URLconf
+    1. Import the include() function: from django.urls import include, path
+    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
+"""
+# from django.contrib import admin
+from django.urls import path
+
+urlpatterns = [
+    #    path('admin/', admin.site.urls),
+]
```

Comment:
```suggestion
from django.urls import path

urlpatterns = [
]
```
주석 처리하여 사용하지 않는 코드는 삭제해주세요 현진님!